### PR TITLE
create .toml file

### PR DESCRIPTION
**Pull request by mdunbavan**
Signed-off-by: Mark Dunbavan <mdunbavan@gmail.com>

### CODE REVIEW

**Review by naveensrinivasan — APPROVED** [review]

File: netlify.toml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# Default build command.
+command = "nuxt build"
+
+
+[build.environment]
+
+# Production context: all deploys from the Production branch set in your site's
+# deploy contexts will inherit these settings.
+[context.main.environment]
+REPO_URL="https://github.com/ossf/scorecard"
```
Comment (azeemshaikh38) [inline netlify.toml:23]: Should this be scorecard-webapp?

**Comment by azeemshaikh38** [comment]
Could you add instructions on how changes to the netlify.toml should be deployed? 

Also, instructions on deploying code changes would be helpful. I assume every time we make a change within scorecards-site, we need to rebuild dist directory and commit that to the repo too?

**Comment by mdunbavan** [comment]
> Could you add instructions on how changes to the netlify.toml should be deployed?
> 
> Also, instructions on deploying code changes would be helpful. I assume every time we make a change within scorecards-site, we need to rebuild dist directory and commit that to the repo too?

Hi Azeem.

Any changes to the .toml file just get committed and Netlify will know right away and make changes to the server.
Any changes to the code will be built by netlify automatically and statically generated so you don't do anything at all.
Make sure you make use of deploy previews in GitHub when making changes to the codebase.

File: netlify.toml (hunk 1 above)
Comment (mdunbavan) [reply netlify.toml]: Added in commit

**Comment by mdunbavan** [comment]
> Could you add instructions on how changes to the netlify.toml should be deployed?
> 
> Also, instructions on deploying code changes would be helpful. I assume every time we make a change within scorecards-site, we need to rebuild dist directory and commit that to the repo too?

The reason there was an issue in deploying first time round was that when the site was created in netlify it would have built the dist folder in the root dir not scorecards-site dir. When you initially set the site up there is usually an option before you init the server being created on where the publish and base dir will be and my guess is that this was done once the server had been created.

**Review by azeemshaikh38 — COMMENTED** [review]
Thanks!

**Pull request merged**